presto-spi/pom.xml

```diff
@@ -88,7 +88,7 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <scope>test</scope>
+            <scope>provided</scope>
```

Comment:
We intentionally don't require guava in the spi. we want it to have as few external dependencies as possible.